react/Field/index.jsx

```diff
@@ -246,7 +246,8 @@ const allowedTypes = [
   'textarea',
   'text',
   'url',
-  'number'
+  'number',
+  'contact'
```

Comment:
Is that a real input type? I've never heard of it and a quick search doesn't return anything.

Reply:
Yes it is real :) https://github.com/cozy/cozy-ui/blob/master/react/Field/index.jsx#L182